src/views/Player.vue

```diff
@@ -74,9 +74,12 @@ var timeout = null;
 
 // wait this much time (secs) then show error page
 // if browser is not supported
-
 var browserCheckTime = 10;
 
+// how many seconds to step back (currentTime) when a user comes back to
+// a plio for a new session
+var stepBackTime = 5;
+
```

Comment:
This means if a user watched till time `x` in a session, and they come back to a new session, the video will start playing for them from `x-5` seconds

Reply:
`var` nahi `let` use karo, since it's a constant

Reply:
But in all of our constants we have used `var`.

<img width="455" alt="Screenshot 2021-01-07 at 12 13 35 PM" src="https://user-images.githubusercontent.com/20169829/103860529-c6511b80-50e1-11eb-99a4-fa1d88a1ee55.png">

Also if you see (here)[https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var], `var` is better suited for global use and `let` is more suited for scoping in a block use